gui/README.md

```diff
@@ -1 +1,58 @@
-# GUI for RespiraWorks open source ventilator.
+# Ventilator GUI
+
+Here resides the code for the ventilator graphical user interface.
+
+## QtCreator
+
+We're using QtCreator to create the qml layout file (main.qml) that is used to
+generate the graphical user interface. Note that any ressources (E.g. Images,
+other qml files) need to be referenced in the qml.qrc file.
+
+## Installing Qt Dependencies
+
+To install the necessary Qt dependencies, use:
```

Comment:
On Mac these things will be different but I (or someone else) can edit it later.